src/main/java/model/data/DataTable.java

```diff
@@ -25,7 +27,7 @@ public DataTable() {
 	 */
 	public DataTable(String name, List<DataRow> rows, List<DataColumn> columns) {
 		this();
-		this.rows = new ArrayList<>(rows);
+		this.rows = rows;
```

Comment:
Mijn methode was toch mooier. Nu weet je niet of rows wel of niet immutable is, als je een kopie maakt, zit je veel veiliger.